Classes/Refactory/Login/Controller/LoginController.php

```diff
@@ -53,7 +53,7 @@ protected function initializeAction()
     {
         # Set locale based on Accept-Language
         $detector = new \Neos\Flow\I18n\Detector();
-        $acceptLanguageHeader = $this->request->getHttpRequest()->getHeaders()->get('Accept-Language');
+        $acceptLanguageHeader = $this->request->getHttpRequest()->getHeaderLine('Accept-Language');
```

Comment:
Is this available in all Neos versions?

If not please update the minimal requirement in `composer.json`.

Thanks ;-)